### ISSUE TITLE
Extend MetaEditorTitleColumn class

### ISSUE THREAD

**Issue by jakmax76** [axllent/silverstripe-meta-editor#2]
Hi,
i tried to extend MetaEditorTitleColumn class
to override method **getErrorMessages()** because it's not translatable. 
I tried to use 

```
SilverStripe\Core\Injector\Injector:
  Axllent\MetaEditor\Forms\MetaEditorTitleColumn:
    class: CustomMetaEditorTitleColumn

```

(my class extends MetaEditorTitleColumn)

```
class CustomMetaEditorTitleColumn extends MetaEditorTitleColumn
{
  public function getErrorMessages()
      { .... }
}
```

but it does'nt work

i wrong something? Can you help me?

Thank you

**Comment by axllent** [comment]
Hi there. I'm not actually sure how to solve this issue this way. I think I should just update the code to allow those error messages to be translatable (and probably other areas), but I'd need to find some time to do that (which I don't have right now).

Translatable allows variables to be used, an example from somewhere in the CMS is:
```
_t(
    self::class . '.PAGETYPE_TITLE',
    '(Page type: {type}) {title}',
    [
        'type' => $node->i18n_singular_name(),
        'title' => $node->Title,
    ]
)
```
If you had time and were able, I would appreciate a pull request to add this functionality? Otherwise leave this with me and I'll look into it when I find some time, thanks.

**Comment by axllent** [comment]
Fantastic, thank you, that really helps! I have merged your PR and made a couple of minor edits, namely to the key names which you can see [here](https://github.com/axllent/silverstripe-meta-editor/blob/master/lang/en.yml) (just to shorten some of them a bit). This has been released as 1.2.2. Please feel free to re-open this if you have issues.
